app_feup/lib/view/Widgets/bug_report_form.dart

```diff
@@ -91,16 +88,27 @@ class BugReportFormState extends State<BugReportForm> {
       bottomMargin: 30.0,
     ));
 
+    formWidget.add(FormTextField(
+      emailController,
+      Icons.mail,
+      minLines: 1,
+      maxLines: 2,
+      description: 'Contacto (opcional)',
+      labelText: 'Email em que desejas ser contactado',
+      hintText: 'Informação pública na página do GitHub!',
```

Comment:
Maybe we should make this more explicit to prevent any legal issues or misunderstandings. Can we add a dialog box upon submission of the bug (whether the user provided email address or not) saying something like "A informação submetida será disponibilizada publicamente na plataforma GitHub" with two buttons "Cancelar" and "Aceito"?

Reply:
Maybe it would be even simpler to have that text in a checkbox above the send button, and disallow sending if the checkbox isn't checked, with an error message?

Reply:
That would work also yes. If it doesn't clutter the screen too much, it works for me!